code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -102,14 +102,30 @@
 	icon_state = "c20r[magazine ? "-[Ceiling(get_ammo(0)/4)*4]" : ""][chambered ? "" : "-e"][silenced ? "-suppressed" : ""]"
 	return
 
+/obj/item/weapon/gun/projectile/automatic/wt550
+	name = "security auto rifle"
+	desc = "An outdated personal defence weapon. Uses 4.6x30mm rounds and is designated the WT-550 Automatic Rifle."

```

Comment:
"An outdated personal defense weapon, the WT-550 Automatic Rifle fires 4.6x30mm rounds, and was typically used in law enforcement."